akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -109,6 +109,8 @@ Java
 We start an Actor system from the defined `main` behavior and send two `Start` messages that
 will kick-off the interaction between two separate `bot` actors and the single `greeter` actor.
 
+A normal application consists of a single `ActorSystem` which in turn contains many actors. 
```

Comment:
a single ActorSystem per JVM?

I'm thinking about a clustered application, which consists of many JVMs

Reply:
Good point, I'll change that